cron/getIncrementBalance.js

```diff
@@ -0,0 +1,29 @@
+const cron = require('node-cron');
+const { familyModel } = require('../database/models');
+
+const { getLogger } = require('../helpers');
+const logger = getLogger('getIncrementBalance');
+
+async function main() {
+  try {
+    await cron.schedule('1 * * * *', async function () {
+
+      const families = await familyModel.find({});
+
+      families.forEach((item) => {
+        item.balance = item.totalSalary + item.passiveIncome;
```

Comment:
Your balance should not be equal totalSalary + passiveIncome. You'll loose savings for previous months then